packages/contracts/test/contracts/L1/rollup/CanonicalTransactionChain.spec.ts

```diff
@@ -135,6 +136,69 @@ describe('CanonicalTransactionChain', () => {
     )
   })
 
+  describe('Gas param setters', () => {
+    describe('setGasDivisor', async () => {
+      it('should revert when not called by the sequencer', async () => {
```

Comment:
Out of date description here

```suggestion
      it('should revert when not called by the Burn Admin', async () => {
```